frontend/app-development/features/SettingsModalButton/SettingsModal/components/Tabs/AbouTab/AboutTab.module.css

```diff
@@ -1,3 +1,47 @@
 .contentWrapper {
   max-width: 500px;
 }
+
+.createdFor {
+  display: flex;
+  align-items: center;
+  background-color: var(--fds-semantic-surface-neutral-subtle);
+  border-radius: 4px;
+  padding-inline: 12px;
+  padding-block: 20px;
+  margin-top: 10px;
+}
+
+.createdBy {
+  display: flex;
+  align-items: center;
+  margin-block: 10px;
+  color: var(--fds-semantic-text-action-active);
+}
+
+.label {
+  margin-top: 20px;
+}
```

Comment:
Hva tenker du om å fjerne denne? Ser ut som vi allerede bruker spacing-propen på label komponenten. 😊 
```suggestion
```

Reply:
Tenker vi beholder denne? 
Spacing tar vel å legger på space under komponenten slik jeg har skjønt det, mens i dette tilfellet så ønsker vi spacing på toppen av labelen 😄

Reply:
ah, sorry - kunne vi heller lagt til margin-bottom på elementet som er over istedenfor? 😊 Så slipper vi å forholde oss til både bottom om top? 🤔

Reply:
Ja, det kan sikkert gjøres. 
Men, da må vi skrive ```margin-bottom: 20px``` to ganger istedenfor en, siden ```.label``` er brukt flere steder der blokken over er forskjellig. 
```.label``` er brukt rett etter ```.contentWrapper``` (som er brukt to steder, og skal bare ha margin-bottom: 20px på det ene stedet), og rett etter ```.createdFor```. 
Tror helt ærlig det beste er å beholde det som det er.